edge-app-template/src/clj/new/edge_app_template.clj

```diff
@@ -22,16 +22,23 @@
   [name & opts]
   (binding [*dir* (sanitize-ns name)]
     (let [opts (set (map #(keyword (subs % 2)) opts))
+          cljs? (contains? opts :cljs)
+          sass? (contains? opts :sass)
+          reagent? (contains? opts :reagent)
           data {:name (project-name name)
                 :sanitized (name-to-path name)
                 :root-ns (multi-segment (sanitize-ns name))
-                :sass (contains? opts :sass)
-                :cljs (contains? opts :cljs)
-                :kick (or (contains? opts :sass)
-                          (contains? opts :cljs))
+                :sass sass?
+                :cljs cljs?
+                :reagent reagent?
```

Comment:
can we rename to reframe? this is very much a reframe template, not a reagent one.